### PR TITLE
Change sample.sequenced_at to reads_updated_at

### PR DESCRIPTION
**Pull request by islean**
## Description

Addresses issue #2388.

### Changed

- sample.sequenced_at is now called sample.reads_updated_at.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b sequenced-at-to-reads-updated-at -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by IO
- [ ] "Merge and deploy" approved by VJ, SD
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
Nevermind, I did not wait long enough... seems good though

image

**Comment by islean** [comment]
Downgrade worked:

image

**Comment by islean** [comment]
Tested:

`cg get sample` with success.
`θ62° [isak.ohlsson@hasta:~] [S_base] 20s 1 $ cg add sample -s male -a WGSPCFC030 -p standard cust000 test_sample
hardlyfastminnow: new sample added`

`θ65° [isak.ohlsson@hasta:~] [S_base] 20s $ cg transfer lims --sample-id test_sample
0 samples to process`

`cg workflow mip-dna start-available` found cases and started handling them.

`cg workflow mip-dna start <case_id>` found the case and started running a command towards scout.

`cg upload auto --pipeline mip-dna` found the cases although the actual upload failed due to failed prior uploads.

`cg upload -c <case_id>` went through fine apart from a sample not existing in Lims.

`cg deliver analysis -c <case_id> -d fastq -d mip-dna` did not link due to no samples having enough reads.

**Review by Vince-janv — APPROVED** [review]
Big diff, but I couldn't spot anything out of the ordinary. Well done 👍

File: cg/meta/transfer/flowcell.py
Comment (diitaz93) [inline cg/meta/transfer/flowcell.py:1]: What is the purpose of this file?

File: tests/meta/transfer/test_meta_transfer_flowcell.py
Comment (diitaz93) [inline tests/meta/transfer/test_meta_transfer_flowcell.py:1]: also this file?

**Review by diitaz93 — APPROVED** [review]
Sorry for the delay, looks good 🌟

File: cg/meta/transfer/flowcell.py
Comment (islean) [reply cg/meta/transfer/flowcell.py:1]: Oops some artifact from a merge conflict!

File: tests/meta/transfer/test_meta_transfer_flowcell.py
Comment (islean) [reply tests/meta/transfer/test_meta_transfer_flowcell.py:1]: Removed this as well!

**Comment by islean** [comment]
Deployment to hasta stage and cg-vm1 successful: 

image

**Comment by islean** [comment]
Downgrade also successful.

**Comment by islean** [comment]
Deployed to stage and production 2023-09-25 by IO.

**Pull request merged**